js/views/pageNavVw.js

```diff
@@ -238,6 +238,17 @@ module.exports = baseVw.extend({
       notif.type = notif.type || notif.message_type;
       notifStamp = Date.now();
       notif.guid = notif.guid || notif.sender;
+
```

Comment:
Even with this approach, the counts will still be off, because when the user lazy loads a new page of notifications, the returned count includes the dupes. You could keep track of yet another offset to account for them, but if I'm remembering correctly we are already keeping track of 3 offsets and it took a lot of tinkering to account for all the edge cases. If you do want to add another offset, please do so at your own risk!

FWIW, IMHO this should really be the serving preventing too frequent dupes on the follow POST so the dupes don't even make it into the DB and therefore never affect the count.